node/src/types/chainspec.rs

```diff
@@ -81,6 +81,14 @@ impl Chainspec {
         }
 
         let min_era_ms = 1u64 << self.highway_config.minimum_round_exponent;
+
+        if self.core_config.unbonding_delay < self.core_config.auction_delay {
```

Comment:
If they're equal and I request unbonding in era `N`, I might still be a validator in era `N + auction_delay`, and I'd get my money back right at the end of that era, right?
That still seems a bit risky to me; maybe we should print a warning in that case, as well, or even disallow it?

Reply:
I'd favour disallowing it.  Not sure if there's any advantage to allowing them to be equal?